pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -28,6 +28,8 @@ const (
 	// This should remain in the default role bindings for the NodeAdmin role
 	LegacyMasterKubeletAdminClientUsername = "system:master"
 	MasterKubeletAdminClientUsername       = "system:openshift-node-admin"
+
+	InfraNodeBootstrapServiceAccountName = "system:" + NodeBootstrapRoleName
```

Comment:
the constant is `InfraNodeBootstrapServiceAccountName`, but `system:node-bootstrapper` is not the name of the service account